### PR TITLE
Add jitter to the RDM response times.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/transceiver.c [hunk 1]
```diff
@@ -403,10 +404,12 @@ static inline void PrepareRDMResponse() {
...
-  // Enable timer
-  // TODO(simon): Use rdm_responder_delay & rdm_responder_jitter to make this
-  // configurable.
-  PLIB_TMR_Period16BitSet(TMR_ID_3, 1760 - RESPONSE_FUDGE_FACTOR);
+  // Enable the timer to trigger when we send the RDM response.
+  unsigned int jitter = 0;
+  if (g_timing_settings.rdm_responder_jitter) {
+    jitter = Random_PseudoGet() % g_timing_settings.rdm_responder_jitter;
+  }
+  PLIB_TMR_Period16BitSet(TMR_ID_3, 1760 - RESPONSE_FUDGE_FACTOR + jitter);
```
Comment (peternewman) [inline firmware/src/transceiver.c]: Can we make the 1760 a const for now until its configurable?

**Comment by peternewman** [comment]
LGTM

File: firmware/src/transceiver.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/transceiver.c]: Actually that's a bug. Give me a sec to fix.

**Comment by peternewman** [comment]
LGTM

**Comment by peternewman** [comment]
LGTM

**Pull request merged**